src/libcore/result.rs

```diff
@@ -501,6 +501,8 @@ impl<T, E> Result<T, E> {
 
     /// Returns an iterator over the possibly contained value.
     ///
+    /// The iterator yields one value if the result is [`Ok`], otherwise none.
```

Comment:
No problem, I could have been confused myself so I appreciate you checking.

I can send you a little cleanup pr later to linkify other instances.  It sure would be nice if we can make it automatic later.